### ISSUE TITLE
Callback for monitoring & controlling a training process using a Telegram bot

### ISSUE THREAD

**Issue by eyalzk** [keras-team/keras#12134]
Hi!
I have developed a nice feature that I use with Keras and thought it could be a useful addition to the official API.
It allows getting automatic loss & metrics updates to the Telegram app using a dedicated callback.
Features include:

  - Get per epoch updates on the loss, accuracy etc.
  - Change the learning rate
  - Get loss convergence plots
  - Kill the training process
  - Query the latest LR or metrics
  - Limit access to a specific Telegram user id

Full details available in [this repo](https://github.com/eyalzk/telegrad) and this [post](https://towardsdatascience.com/how-to-monitor-and-control-deep-learning-experiments-through-your-phone-35ef1704928d)

Usage depends on the [telegram-pyhton-api](https://github.com/python-telegram-bot/python-telegram-bot). However, dependencies in callbacks are not new (exist for example in RemoteMonitor callback).

Including this in Keras will require some changes to my implementation, which I will happily do, but I wanted to check if this is something that people would want before making any adjustments.

**Usage**:
If approved, I will merge the bot class into the callback class. Then, usage will look something like this:
```python
#  imports
from Keras.callbacks import TelegramBotCallback

telegram_token = "TOKEN"  # Replace TOKEN with your bot's token

#  User id is optional, however highly recommended as it limits the access to you alone.
telegram_user_id = None  # integer

# Create a TelegramBotCallback instance
telegram_callback = TelegramBotCallback(token=telegram_token, user_id=telegram_user_id)

# Build your model...

# Add telegram_callback to the list of callbacks passed to model.fit:
model.fit(x_train, y_train,
          batch_size=batch_size,
          epochs=epochs,
          verbose=1,
          validation_data=(x_test, y_test),
          callbacks=[telegram_callback])
```

And your bot will automatically start interacting with the training process.
Working example and screenshots are available in the [Telegrad repo.](https://github.com/eyalzk/telegrad)

So what do you think? Could this be something to be added to Keras?

**Comment by gabrieldemarmiesse** [comment]
We cannot support all features possible in keras due to the lack of manpower. But you can make a pull request to keras-team/keras-contrib if you accept to maintain this callback in the future :) In keras-contrib, each contributor is responsible for his/her piece of code. Making maintenance much easier for the keras team.

**Comment by eyalzk** [comment]
Thank you for your reply @gabrieldemarmiesse .
I will open an issue in keras-contrib for further discussion. 
If there will be an interest by the maintainers, I will start working on a PR

**Comment by gabrieldemarmiesse** [comment]
We are definitly interested in having it in keras-contrib!
